File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS3003Q10GrdOrderDateListItemInfo.java
package nta.med.data.model.ihis.ocsa;

import java.util.Date;

public class OCS3003Q10GrdOrderDateListItemInfo {
	private Date orderDate;
	private String gwa;
	private String gwaName;
	private String doctorName;
	private String bunho;
	private String doctor;
	private String naewonType;
	private Double jubsuNo;
	private Double pkOrder;
	private String ioGubun;
	private String specificComment;
	private Double pkocskey;
	private String hangmogName;
	private String hangmogCode;
	private String jundalPart;
	private String iraiKubun;
	private String image;
	private String imagePath;
	private Date crTime;
	public OCS3003Q10GrdOrderDateListItemInfo(Date orderDate, String gwa,
			String gwaName, String doctorName, String bunho, String doctor,
			String naewonType, Double jubsuNo, Double pkOrder, String ioGubun,
			String specificComment, Double pkocskey, String hangmogName,
			String hangmogCode, String jundalPart, String iraiKubun,
			String image, String imagePath, Date crTime) {
		super();
		this.orderDate = orderDate;
		this.gwa = gwa;
		this.gwaName = gwaName;
		this.doctorName = doctorName;
		this.bunho = bunho;
		this.doctor = doctor;
		this.naewonType = naewonType;
		this.jubsuNo = jubsuNo;
		this.pkOrder = pkOrder;
		this.ioGubun = ioGubun;
		this.specificComment = specificComment;
		this.pkocskey = pkocskey;
		this.hangmogName = hangmogName;
		this.hangmogCode = hangmogCode;
		this.jundalPart = jundalPart;
		this.iraiKubun = iraiKubun;
		this.image = image;
		this.imagePath = imagePath;
		this.crTime = crTime;
	}
	public Date getOrderDate() {
		return orderDate;
	}
	public void setOrderDate(Date orderDate) {
		this.orderDate = orderDate;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public String getNaewonType() {
		return naewonType;
	}
	public void setNaewonType(String naewonType) {
		this.naewonType = naewonType;
	}
	public Double getJubsuNo() {
		return jubsuNo;
	}
	public void setJubsuNo(Double jubsuNo) {
		this.jubsuNo = jubsuNo;
	}
	public Double getPkOrder() {
		return pkOrder;
	}
	public void setPkOrder(Double pkOrder) {
		this.pkOrder = pkOrder;
	}
	public String getIoGubun() {
		return ioGubun;
	}
	public void setIoGubun(String ioGubun) {
		this.ioGubun = ioGubun;
	}
	public String getSpecificComment() {
		return specificComment;
	}
	public void setSpecificComment(String specificComment) {
		this.specificComment = specificComment;
	}
	public Double getPkocskey() {
		return pkocskey;
	}
	public void setPkocskey(Double pkocskey) {
		this.pkocskey = pkocskey;
	}
	public String getHangmogName() {
		return hangmogName;
	}
	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getJundalPart() {
		return jundalPart;
	}
	public void setJundalPart(String jundalPart) {
		this.jundalPart = jundalPart;
	}
	public String getIraiKubun() {
		return iraiKubun;
	}
	public void setIraiKubun(String iraiKubun) {
		this.iraiKubun = iraiKubun;
	}
	public String getImage() {
		return image;
	}
	public void setImage(String image) {
		this.image = image;
	}
	public String getImagePath() {
		return imagePath;
	}
	public void setImagePath(String imagePath) {
		this.imagePath = imagePath;
	}
	public Date getCrTime() {
		return crTime;
	}
	public void setCrTime(Date crTime) {
		this.crTime = crTime;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS3003Q10GrdOrderDateListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OCS3003Q10GrdOrderDateListItemInfo
	{
		private String _orderDate;
		private String _gwa;
		private String _gwaName;
		private String _doctorName;
		private String _bunho;
		private String _doctor;
		private String _naewonType;
		private String _jubsuNo;
		private String _pkOrder;
		private String _ioGubun;
		private String _specificComment;
		private String _pkocskey;
		private String _hangmogName;
		private String _hangmogCode;
		private String _jundalPart;
		private String _iraiKubun;
		private String _image;
		private String _imagePath;
		private String _crTime;

		public String OrderDate
		{
			get { return this._orderDate; }
			set { this._orderDate = value; }
		}

		public String Gwa
		{
			get { return this._gwa; }
			set { this._gwa = value; }
		}

		public String GwaName
		{
			get { return this._gwaName; }
			set { this._gwaName = value; }
		}

		public String DoctorName
		{
			get { return this._doctorName; }
			set { this._doctorName = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String Doctor
		{
			get { return this._doctor; }
			set { this._doctor = value; }
		}

		public String NaewonType
		{
			get { return this._naewonType; }
			set { this._naewonType = value; }
		}

		public String JubsuNo
		{
			get { return this._jubsuNo; }
			set { this._jubsuNo = value; }
		}

		public String PkOrder
		{
			get { return this._pkOrder; }
			set { this._pkOrder = value; }
		}

		public String IoGubun
		{
			get { return this._ioGubun; }
			set { this._ioGubun = value; }
		}

		public String SpecificComment
		{
			get { return this._specificComment; }
			set { this._specificComment = value; }
		}

		public String Pkocskey
		{
			get { return this._pkocskey; }
			set { this._pkocskey = value; }
		}

		public String HangmogName
		{
			get { return this._hangmogName; }
			set { this._hangmogName = value; }
		}

		public String HangmogCode
		{
			get { return this._hangmogCode; }
			set { this._hangmogCode = value; }
		}

		public String JundalPart
		{
			get { return this._jundalPart; }
			set { this._jundalPart = value; }
		}

		public String IraiKubun
		{
			get { return this._iraiKubun; }
			set { this._iraiKubun = value; }
		}

		public String Image
		{
			get { return this._image; }
			set { this._image = value; }
		}

		public String ImagePath
		{
			get { return this._imagePath; }
			set { this._imagePath = value; }
		}

		public String CrTime
		{
			get { return this._crTime; }
			set { this._crTime = value; }
		}

		public OCS3003Q10GrdOrderDateListItemInfo() { }

		public OCS3003Q10GrdOrderDateListItemInfo(String orderDate, String gwa, String gwaName, String doctorName, String bunho, String doctor, String naewonType, String jubsuNo, String pkOrder, String ioGubun, String specificComment, String pkocskey, String hangmogName, String hangmogCode, String jundalPart, String iraiKubun, String image, String imagePath, String crTime)
		{
			this._orderDate = orderDate;
			this._gwa = gwa;
			this._gwaName = gwaName;
			this._doctorName = doctorName;
			this._bunho = bunho;
			this._doctor = doctor;
			this._naewonType = naewonType;
			this._jubsuNo = jubsuNo;
			this._pkOrder = pkOrder;
			this._ioGubun = ioGubun;
			this._specificComment = specificComment;
			this._pkocskey = pkocskey;
			this._hangmogName = hangmogName;
			this._hangmogCode = hangmogCode;
			this._jundalPart = jundalPart;
			this._iraiKubun = iraiKubun;
			this._image = image;
			this._imagePath = imagePath;
			this._crTime = crTime;
		}

	}
}